source/common/quic/envoy_quic_client_connection.cc

```diff
@@ -69,7 +69,27 @@ void EnvoyQuicClientConnection::processPacket(
       quic::QuicTime::Delta::FromMicroseconds(
           std::chrono::duration_cast<std::chrono::microseconds>(receive_time.time_since_epoch())
               .count());
+  ASSERT(peer_address != nullptr && buffer != nullptr);
   ASSERT(buffer->getRawSlices().size() == 1);
+  if (local_address == nullptr) {
+    // Quic doesn't know how to handle packets without destination address. Drop them here.
```

Comment:
nit: The problem is `local_address` being null, which is "source address" not "destination address", right? (`peer_address` would be "destination address", I think).

Here, and elsewhere, including the PR description.

Reply:
It is the destination address of the incoming packet.

Reply:
Oh good lord! Yes, you're right, of course. Sorry!